src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(person =>
+    (Number.isInteger(century))
+      ? (person.sex === 'm'
+      && Math.ceil(person.died / 100) === century)
+      : person.sex === 'm');
+
+  const agedSum = getAverageAge(men);
+
+  return agedSum / men.length;
+}
+
+function getAverageAge(people) {
+  return people.reduce((prev, { born, died }) =>
+    prev + died - born,
+  0);
 }
```

Comment:
But this function calculates sum of ages 🙃

Reply:
`getAgesSum` will be better

on real projects usually we using a lot of functions that was created by other people, for that case we'll expect that our function will return average value ('caused it called getAverage), but it's result - is sum, we'll spent a lot of time to understand whats wrong in it

It's another point why variables naming is important